x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/services/analytics_service/get_analytics.ts

```diff
@@ -143,7 +140,7 @@ export const getAnalyticsFactory = (
               checkpointing: {},
               config,
               id: config.id,
-              job_type: Object.keys(config.analysis)[0] as ANALYSIS_CONFIG_TYPE,
+              job_type: Object.keys(config.analysis)[0] as DataFrameAnalyticsType,
```

Comment:
We have `getAnalysisType()` available in `plugins/ml/public/application/data_frame_analytics/common/analytics.ts`.

Reply:
Updated here cc8f5dd